include/zephyr/xen/dom0/domctl.h

```diff
@@ -12,27 +12,214 @@
 
 #include <zephyr/kernel.h>
 
+/**
+ * @file
+ *
+ * @brief Xen Domain Control Interface
+ */
+
```

Comment:
if you check other generic headers - "file header" is added as first block in the file or right after copyright block, can we follow the same, pls.